src/openapi.yaml

```
@@ -18,6 +18,24 @@ paths:
 
   # TODO: add paths here
 
+  # /users endpoints
+  /users/{user_id}/entries:
+    $ref: "./paths/users/user_id/entries.yaml"
+
+  /users/{user_id}/entries/{entry_id}:
+    $ref: "./paths/users/user_id/entries/entry_id/entry.yaml"
+
+  /users/{user_id}/reviews:
+    $ref: "./paths/users/user_id/reviews.yaml"
+
+  /users/{user_id}/reviews/{review_id}:
+    $ref: "./paths/users/user_id/reviews/review_id/review.yaml"
+
+  /users/{user_id}:
+    $ref: "./paths/users/user_id/account.yaml"
```

```suggestion
    $ref: "./paths/users/user_id/user.yaml"
```
Please rename this file for consistency.